database/README.md

```diff
@@ -45,6 +45,7 @@ import { useList } from 'react-firebase-hooks/database';
 
 const DatabaseList = () => {
   const [snapshots, loading, error] = useList(firebase.database().ref('list'));
+  const value = snapshots.val()
```

Comment:
I think it's more complex than this. `useList` returns an array of firebase snapshot (line 37).

The correct fix is to change `value` on line 55 and 59 to `snapshots`.